config.py

```diff
@@ -20,6 +20,7 @@ class Config(object):
     GOOGLE_ANALYTICS = os.getenv('GOOGLE_ANALYTICS', None)
     NON_DEFAULT_VARIABLES = ['SECRET_KEY', 'SECURITY_USER_NAME', 'SECURITY_USER_PASSWORD', 'JWT_SECRET']
     AVAILABILITY_BANNER = os.getenv('AVAILABILITY_BANNER', False)
+    SECURE_COOKIES = os.getenv('SECURE_COOKIES', True)
```

Comment:
When running locally on Docker, the secure flag isn't set for me on the session cookie.

Reply:
I think the getting of booleans from the environment needs to be looked at. The only way to get this to evaluate to `False` is to set the env var to empty, which doesn't seem possible in the docker-compose setup. This should evaluate environment variable booleans in a similar way to survey runner (https://github.com/ONSdigital/eq-survey-runner/blob/master/app/settings.py#L15).